apis/health.go

```diff
@@ -0,0 +1,29 @@
+package apis
+
+import (
+	"github.com/labstack/echo/v5"
+	"github.com/pocketbase/pocketbase/core"
+	"net/http"
+)
+
+// bindHealthApi registers the health api endpoint.
+func bindHealthApi(app core.App, rg *echo.Group) {
+	api := healthApi{app: app}
+
+	subGroup := rg.Group("/health")
+	subGroup.GET("", api.healthCheck)
+}
+
+type healthApi struct {
+	app core.App
+}
+
+// healthCheck returns a 200 OK response if the server is healthy.
+func (api *healthApi) healthCheck(c echo.Context) error {
+	payload := map[string]any{
+		"code":    http.StatusOK,
+		"status":  "ok",
```

Comment:
If `status` is not required by some 3rd party tool, let's minimize the returned fields and remove it from the response since it could change in the future (the `code` and 200 response should be enough indication for healthy state).

Reply:
Done 👍